packages/react-components/react-shared-contexts/stories/UseAnnouce/UseAnnounceDescription.md

```diff
@@ -0,0 +1,56 @@
+`useAnnounce()` is a React hook that provides a function that can be used to announce messages to screen readers.
+
+**Note:** This hook requires that you app has injected the implementation of announcements into the DOM and announce function is passed to `<AnnounceProvider />`:
```

Comment:
```suggestion
**Note:** This hook requires an aria-live announcer implementation that is configured through the  `<AnnounceProvider />`. Define this context near the top level of your application.:
```

Reply:
aria-live is a pretty terrible thing to implement. I do think that a standard implementation of an announcer should be offered by us. It's what I meant that we could port the FAI one over to FUI and make FAI depend on FUI there

Reply:
I will port an announcer in a separate PR, I decided to not mention it there before it will be done.